flixel/text/FlxText.hx

```diff
@@ -890,7 +890,7 @@ class FlxText extends FlxSprite
 		if (oldWidth != newWidth || oldHeight != newHeight)
 		{
 			// Need to generate a new buffer to store the text graphic
-			var key:String = FlxG.bitmap.getUniqueKey("text");
+			var key:String = FlxG.bitmap.getUniqueKey('text(${this.text})');
```

Comment:
can you talk about the issue this change solved, also, just made this https://github.com/HaxeFlixel/flixel/issues/3081